install_config/configuring_authentication.adoc

```diff
@@ -128,81 +128,132 @@ challenges should set this header to a non-empty value.
 Not supported by all identity providers.
 
 If you want users to be sent to a branded page before being redirected to
-the identity provider's login, then set `*oauthConfig -> alwaysShowProviderSelection: true*`
-in the master configuration file. This provider selection page can be
+the identity provider's login, then set `*oauthConfig -> alwaysShowProviderSelection: true*`. This provider selection page can be
 xref:../install_config/web_console_customization.adoc#customizing-the-login-page[customized].
 
 |`mappingMethod`  | Defines how new identities are mapped to users when they login. See xref:mapping-identities-to-users[Mapping Identities to Users] for more information.
 |===
 
-[NOTE]
 When adding or changing identity providers, you can map identities from the new
 provider to existing users by setting the `*mappingMethod*` parameter to
 `*add*`.
 
+After saving changes to the master configuration file, you must restart the {product-title} services for the changes to take effect:
```

Comment:
Good question. I am not a typical user and had trouble with setting an identity because I did not restart, as the step was not in the docs. Is this something our typical users would know to do? I was asked to make that requirement "more explicit" on the page. 
I was thinking that the user might jump right to the section he wants, say HTPasswd, and not see the restart requirement if not right there in the procedure. 
It does seem redundant, but only when reading through all of the provider types.